README.md

```diff
@@ -235,10 +235,10 @@ From the [Yoti Hub](https://hub.yoti.com):
 To run the Flask or Django container:
  
 1. Clone this repository
-1. Change directory to the example project directory with
-   * `cd examples/yoti_example_flask` for __Flask__   
+1. Rename one of the provided Dockerfiles
+   * `cp flask.dockerfile Dockerfile` for __Flask__
    __OR__
-   * `cd examples/yoti_example_django` for __Django__
+   * `cp django.dockerfile Dockerfile` for __Django__
```

Comment:
We could instead update the context in the `docker-compose.yml` files like this:
```yml
services:
  web:
    build:
      context: ../../
      dockerfile: examples/yoti_example_django/Dockerfile
```